docs/src/examples/block-explorer.md

```diff
@@ -1,10 +1,10 @@
 # Block Explorer
 
-An extremely basic block explorer backend implementation that shows how to leverage basic Fuel indexer abstractions in order to build a cool dApp backend.
+An rudimentary block explorer backend implementation demonstrating how to leverage basic Fuel indexer abstractions in order to build a cool dApp backend.
```

Comment:
Consider:
> A rudimentary...

Reply:
Yea I thought that was the change that went in the docs PR? Not sure how it got changed back. Anyway, will make this change